editor.Rmd

```diff
@@ -149,52 +147,54 @@ Tag the account(s) of all post authors and package authors and reviewers.
 If not already included in their author metadata, you can search for their account but only tag them if
 
 * it is active,
-* it is not anonymous,
+* it is not anonymous or private,
 * it is at least partly used professionally by the owner.
 
 When in doubt, use the person's name, or time permitting, contact the person whose account you'd like to tag.
 
-For a package wrapping a service present on Twitter you can tag that account (e.g. for a tweet about `rredlist` that accesses IUCN Red List you might tag IUCN Twitter account).
+For a package wrapping a service present on the social media you can tag that account (e.g. for a Mastodon post about `rredlist` that accesses IUCN Red List you might tag IUCN Mastodon account).
 
 When tagging accounts include them in a sentence e.g. "Thanks to `@account1`"/ "As told by `@account2`". 
-We do not use tagging in tweets to ask for attention (i.e. no account names used like hashtags at the bottom of a tweet) because it could be viewed as spamming mentioned accounts, and because it creates visual clutter in the tweet.
+We do not use tagging in post to ask for attention (i.e. no account names used like hashtags at the bottom of a post) because it could be viewed as spamming mentioned accounts, and because it creates visual clutter in the post.
```

Comment:
```suggestion
We do not use tagging in posts to ask for attention (i.e. no account names used like hashtags at the bottom of a post) because it could be viewed as spamming mentioned accounts, and because it creates visual clutter in the post.
```